src/coreclr/tests/issues.targets

```diff
@@ -395,6 +395,9 @@
 
     <!-- Windows arm32 specific excludes -->
     <ItemGroup Condition="'$(XunitTestBinBase)' != '' and ('$(BuildArch)' == 'arm' or '$(AltJitArch)' == 'arm') and '$(TargetsWindows)' == 'true' and '$(RuntimeFlavor)' == 'coreclr'">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Methodical/explicit/coverage/expl_float_1_*">
```

Comment:
Does that syntax work? You don't need `expl_float_1_*/*` or `expl_float_1_*/**`?

Reply:
Bruce is right, the path should always end with ** as the infra arbitrarily creates an extra subfolder level under the test folder.

Reply:
That's what Nathan also fought when working on Mono-specific test exclusions.